docker/app/dockerfile

```
@@ -0,0 +1,45 @@
+FROM amazoncorretto:21-alpine-jre
+
+LABEL maintainer="UMC Product Team"
+LABEL description="UMC Product Team Official SpringBoot Backend"
+
+# 작업 디렉토리 설정
+WORKDIR /app
+
+# curl 설치 (헬스체크용) + non-root 유저 생성
+RUN apk add --no-cache curl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)